docs/docs/reference/functions/middleware-and-helpers.md

```diff
@@ -133,3 +133,34 @@ export const config: GatsbyFunctionConfig = {
   },
 }
 ```
+
+### How function config is applied
```

Comment:
For now this is just "braindump" of more or less what kind of information I think we should add to the documentation - for now this is just section added, but would be good to figure out proper flow for custom body parser configs (before/after examples, maybe make what I added as one of already existing examples and make it link back to explanation of middlewares priorities?)